screenpipe-vision/src/run_ui_monitoring_macos.rs

```diff
@@ -0,0 +1,77 @@
+use std::process::Stdio;
+use tokio::time::{sleep, Duration};
+use tokio::io::{BufReader, AsyncBufReadExt};
+use tokio::select;
+use tokio::process::Command as TokioCommand;
+use anyhow::Result;
+use log::{info, warn, error, debug};
+
+const UI_MONITORING_SCRIPT: &str = include_str!("ui_monitoring_macos.swift");
```

Comment:
this will not work in the app or need to do some plumbing with packaging files around